server/controllers/Bookhandlers.js

```diff
@@ -42,12 +67,47 @@ exports.allBooks = (req, res) => {
     }
   }).catch(error => res.status(400).send(error)); // catch error from findall
 };
+exports.updateBookQuantity = (req, res) => { // modifybook quantity
+  const bookId = parseInt(req.body.bookId, 10);
+  const bookQuantity = parseInt(req.body.Quantlvl, 10);
+  if (!isNaN(bookId) || !isNaN(bookQuantity)) { // has to be a number really
+    Books.findOne({ // search for book with id
+      where: {
+        id: bookId,
+      },
+    }).then((bookDetails) => {
+      if (bookDetails === null) { // if a book is not found
+        res.status(200).json({
+          status: 'unsuccessful',
+          message: 'Book extint',
+        });
+      } else { // if addQuant is set then add
+        bookDetails.update({
+          bookQuantity:
+            (bookDetails.bookQuantity + bookQuantity) < 1 ? 1 :
+            (bookDetails.bookQuantity + bookQuantity),
+          // Never less than 1
+        }).then(
+          addBook => res.status(200).json({
+            status: addBook,
+          })).catch(error => res.status(400).send(error));
+      }
+    });
+  } else {
+    res.status(400).json({
+      status: 'none',
+      message: 'That\'s not a number',
+    });
+  }
+};
+
```

Comment:
* You are not catching internal server errors. (5XX)
* Use descriptive error messages.